ota-plus-web/app/reactapp-new/src/saga/feedSaga.ts

```diff
@@ -0,0 +1,17 @@
+import { put, takeEvery } from 'redux-saga/effects';
+import { FeedData, actionTypes } from '../store/feed/types';
+import { feedNetworking } from '../networking/FeedNetworking';
+import { Actions } from '../store/feed/actions';
+
+function* getFeedData(action: any) {
```

Comment:
instead of any we can strict type action using action: Extract<Actions, { type: actionTypes.GET_FEED_REQUEST }>